pkg/commands/workload_get.go

```diff
@@ -197,7 +197,7 @@ func (opts *WorkloadGetOptions) Exec(ctx context.Context, c *cli.Config) error {
 
 	// Print workload issues
 	c.Printf("\n")
-	c.EmojiBoldf(cli.SpeechBalloon, "Messages\n")
+	c.Boldf(c.Emoji(cli.SpeechBalloon, "Messages\n"))
```

Comment:
Can this be reversed to avoid any abnormal output on Boldf function because of the rune